documentation/src/docs/asciidoc/release-notes/release-notes-5.8.0-M1.adoc

```diff
@@ -86,6 +86,10 @@ on GitHub.
 * In parameterized tests with `@MethodSource` or `@ArgumentSource`, arguments can now have
   optional names. When the argument is included in the display name of an iteration, this
   name will be used instead of the value.
+* `DynamicTests.stream()` can now consume `Named` input and will use each name-value
+  pair as the display name and value for each generated dynamic test (see
+  link:{userguide-root}++/#writing-tests-dynamic-tests-examples++[User Guide] for details).
```

Comment:
```suggestion
  <<../user-guide/index.adoc#writing-tests-dynamic-tests-examples,User Guide>> for details).
```

Reply:
@marcphilipp beat me to it: I was just about to paste in `(see
  <<../user-guide/index.adoc#writing-tests-declarative-timeouts, User Guide>> for details)` as an example. ;-)

Reply:
Ah yes, the link is targeting the source file directly, got it ^^

Thanks @marcphilipp, and runner-up @sbrannen 😁